### ISSUE TITLE
(mouseEnter) event isn't firing.

### ISSUE THREAD

**Issue by enjoiful** [Wykks/ngx-mapbox-gl#96]
The following code never results in "mouseEnterEvt" function triggering in the component's controller. 

`	<mgl-map
    [style]="'mapbox://styles/enjoiful/cjqlegp1hpigl2rqe8ifg4doc'"
    [zoom]="[13]"
    [center]="[-118.8493, 34.232]"
    (load)="mapLoaded($event)"
    (zoomEvt)="zoomEvt($event)"
    (mouseEnter)="mouseEnterEvt($event)"
  ></mgl-map>`

if you change the (mouseEnter) to (mouseOver), the function gets called correctly.

**Comment by Wykks** [comment]
Indeed. Look like a bug in mapbox-gl itself. I can reproduce it with mapbox-gl only
